mayan/apps/dynamic_search/queues.py

```diff
@@ -50,5 +52,5 @@
     dotted_path='mayan.apps.dynamic_search.tasks.task_saved_resultset_expired_delete',
     label=_(message='Delete expired saved resultsets'),
     name='task_saved_resultset_expired_delete',
-    schedule=timedelta(seconds=TASK_SAVED_RESULTSET_EXPIRED_DELETE_INTERVAL)
```

Comment:
any task with a timedelta is run by mayan's celery beat scheduler. Note that due to a bug with celery beat, we will have to manually remove the scheduled task, as celery is only capable of registering new tasks, not unregistering unused ones.

Reply:
It makes sense. However, this is the only part of the code changes that I find concerning.